File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] arr = new int[2*n];
        for(int i = 0; i < n; i++){
            arr[i*2] = nums[i]; 
        }
        
        for(int i = 1; i < n+1; i++){
            arr[i*2 -1 ] = nums[i+n-1]; 
        }
        return arr;
        
    }
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        p1 = 0
        p2 = n
        ans = []
        while(p2 != len(nums)):
            ans.append(nums[p1])
            ans.append(nums[p2])
            p1 += 1
            p2 += 1
        return ans
